ui/app/src/views/PoolPage.vue

```diff
@@ -66,6 +68,9 @@ export default defineComponent({
         />
       </PoolList>
     </div>
+    <ActionsPanel
+      connectType="connectToAll"
```

Comment:
connectType is `connectToSif` since to Pool you only need to be connected to Sif wallet